src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2185,14 +2189,21 @@ namespace winrt::TerminalApp::implementation
             {
                 co_await winrt::resume_foreground(Dispatcher());
 
+                if (warnMultiLine)
+                {
+                    const auto focusedTab = _GetFocusedTabImpl();
+                    // Do not warn about multi line pasting if the current tab has bracketed paste enabled.
+                    warnMultiLine &= !focusedTab->GetActiveTerminalControl().BracketedPasteEnabled();
```

Comment:
Unfortunately, we can't actually use `&=` for booleans. It's silly, but `&` is only for binary AND.

```suggestion
                    warnMultiLine = warnMultiLine && !focusedTab->GetActiveTerminalControl().BracketedPasteEnabled();
```